Blocks/Persistence/test.cc

```diff
@@ -353,29 +364,35 @@ TEST(PersistenceLayer, FileDirectives) {
 
   using IMPL = current::persistence::File<StorableString>;
 
+  const auto namespace_name = current::ss::StreamNamespaceName("namespace", "top_level_name");
   const std::string persistence_file_name = current::FileSystem::JoinPath(FLAGS_persistence_test_tmpdir, "data");
   const auto file_remover = current::FileSystem::ScopedRmFile(persistence_file_name);
 
   {
     // An empty file - no entries and head equals -1us.
-    IMPL impl(persistence_file_name);
+    IMPL impl(namespace_name, persistence_file_name);
     EXPECT_EQ(0u, impl.Size());
     EXPECT_EQ(-1, impl.CurrentHead().count());
     const auto head_idxts = impl.HeadAndLastPublishedIndexAndTimestamp();
     ASSERT_FALSE(Exists(head_idxts.idxts));
     EXPECT_EQ(-1, head_idxts.head.count());
   }
+  current::reflection::StructSchema struct_schema;
+  struct_schema.AddType<StorableString>();
+  const std::string signature =
+      "#signature\t" + JSON(current::ss::StreamSignature(namespace_name, struct_schema.GetSchemaInfo())) + '\n';
```

Comment:
Initially I just thought that if an entry has the `JSON(idxts_t)\tJSON(entry_t)` format, it would be natural to use `\t` to divide the directive name from its value. But it's really not, I agree.